src/main/java/uk/ac/ebi/eva/pipeline/io/writers/VepAnnotationFileWriter.java

```diff
@@ -32,18 +32,25 @@
 
     private final VepProcess vepProcess;
 
-    public VepAnnotationFileWriter(Long timeoutInSeconds, Integer chunkSize, AnnotationParameters annotationParameters) {
+    private boolean vepProcessOpened = false;
```

Comment:
If you get the open logic from VepProces, something like vepProcess.isOpen() then you can remove the boolean flags / initialization